### ISSUE TITLE
Can't import `constants`

### ISSUE THREAD

**Issue by cmcqueen** [stephane-martin/deescalate#2]
I installed on Linux (Ubuntu 16.04) for Python 3.

```
>>> import deescalate
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.5/dist-packages/deescalate-0.1-py3.5-linux-x86_64.egg/deescalate/__init__.py", line 4, in <module>
    from .main import lockdown_account
  File "/usr/local/lib/python3.5/dist-packages/deescalate-0.1-py3.5-linux-x86_64.egg/deescalate/main.py", line 6, in <module>
    from ._deescalate import py_prctl, C_CapabilitySet, C_BoundingSet
  File "deescalate/_deescalate.pyx", line 6, in init deescalate._deescalate (deescalate/_deescalate.c:2370)
  File "/usr/local/lib/python3.5/dist-packages/deescalate-0.1-py3.5-linux-x86_64.egg/deescalate/utils.py", line 7, in <module>
    from constants import C
ImportError: No module named 'constants'
```

Then I did

    sudo python3 -m pip install constants

Then I got:

```
>>> import deescalate
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.5/dist-packages/deescalate-0.1-py3.5-linux-x86_64.egg/deescalate/__init__.py", line 4, in <module>
    from .main import lockdown_account
  File "/usr/local/lib/python3.5/dist-packages/deescalate-0.1-py3.5-linux-x86_64.egg/deescalate/main.py", line 6, in <module>
    from ._deescalate import py_prctl, C_CapabilitySet, C_BoundingSet
  File "deescalate/_deescalate.pyx", line 6, in init deescalate._deescalate (deescalate/_deescalate.c:2370)
  File "/usr/local/lib/python3.5/dist-packages/deescalate-0.1-py3.5-linux-x86_64.egg/deescalate/utils.py", line 7, in <module>
    from constants import C
ImportError: cannot import name 'C'
```

**Comment by d3dave** [comment]
The original intent was to import from the local constants file, not the constants package. A missing `.`.

**Comment by cmcqueen** [comment]
This was reported nearly two years ago. Can this be fixed? Would you accept a pull request?

**Comment by stephane-martin** [comment]
Should be fixed in 0.1.2.
